ecl/resource_ecl_compute_instance_v2_test.go

```diff
@@ -108,6 +110,62 @@ func TestAccComputeV2Instance_userData(t *testing.T) {
 	})
 }
 
+func TestAccComputeV2Instance_configDrive(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	var instance servers.Server
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeV2InstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeV2InstanceEnabledConfigDrive,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeV2InstanceExists("ecl_compute_instance_v2.instance_1", &instance),
+					resource.TestCheckResourceAttr(
+						"ecl_compute_instance_v2.instance_1", "config_drive", "true"),
+				),
+			},
+		},
+	})
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeV2InstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeV2InstanceDisabledConfigDrive,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeV2InstanceExists("ecl_compute_instance_v2.instance_1", &instance),
+					resource.TestCheckResourceAttr(
+						"ecl_compute_instance_v2.instance_1", "config_drive", "false"),
+				),
+			},
+		},
+	})
+
+	resource.Test(t, resource.TestCase{
```

Comment:
I think you can remove this test because you can confirm that in basic test.

Reply:
Fixed.